mappings/da48ff99a20e8c8235811dded4a83c9f7f307d195b948e1bbab263e5494345.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "da48ff99a20e8c8235811dded4a83c9f7f307d195b948e1bbab263e5494345",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://intraconnection.io/",
```

Comment:
Hi @Intraconnection,

we can't verify the url value. If your page is still not up - you can remove this value, it is not required for the submission. 

Thanks!

Reply:
Addressed. Is it ok now?